packages/components/src/components/input-field/input-field.jsx

```diff
@@ -184,10 +185,13 @@ const InputField = ({
     };
 
     const updateValue = (new_value, is_long_press) => {
-        const formatted_value = format ? format(new_value) : new_value;
+        let formatted_value = format ? format(new_value) : new_value;
         if (is_long_press) {
             setLocalValue(formatted_value);
         } else {
+            if (is_signed && !/^[+-]/.test(formatted_value) && Math.sign(formatted_value) > 0) {
```

Comment:
Will this effect or break other input fields since this is a shared component?

Reply:
It will not break since the whole condition will start evaluation if the user has set `is_signed` to true